lib/compat/wordpress-6.0/block-patterns.php

```diff
@@ -171,6 +171,15 @@ function gutenberg_register_theme_block_patterns() {
 						}
 					}
 
+					// Translate the pattern metadata.
+					$text_domain           = wp_get_theme()->get( 'TextDomain' );
```

Comment:
Looks like we're inside a loop (I guess for child/parent themes), so we should probably use the text domain corresponding to the `$theme` variable?